react/training-project/src/app/components/App/styles.scss

```diff
@@ -0,0 +1,28 @@
+.app {
+  text-align: center;
+}
+
+.app-logo {
+  animation: app-logo-spin infinite 20s linear;
+  height: 80px;
+}
+
+.app-header {
+  background-color: #222;
```

Comment:
Create a file called `colors.scss` and move there all the colors you're going to use. Then, import that file here.